README.md

```diff
@@ -94,7 +94,8 @@ A collection of demos utilizing trestle can be found in the related project [com
 
 ## Development status
 
-Compliance trestle is currently stable and is based on NIST OSCAL version 1.0.4, with active development continuing.
+Compliance trestle is currently stable and is based on NIST OSCAL version 1.0.4, with active development continuing. 1.1.2 version support is about to completed
+and released accordingly, wait for new announcements to be done soon.
 
```

Comment:
@AleJo2995 Should this be updated to say we support v1.1.2 of OSCAL now? because develop is now on v1.1.2 and main will also be on that once we release.